tests/fast-integration/test/fixtures/commerce-mock/index.js

```diff
@@ -728,7 +728,13 @@ async function checkInClusterEventDeliveryHelper(targetNamespace, encoding) {
   await printStatusOfInClusterEventingInfrastructure(targetNamespace, encoding, "lastorder");
 
   // send event using function query parameter send=true
-  await retryPromise(() => axios.post(`https://${mockHost}`, { id: eventId }, { params: { send: true, encoding: encoding } }), 10, 1000)
+  await retryPromise(async () => {
+    const response = await axios.post(`https://${mockHost}`, {id: eventId}, {params: {send: true, encoding: encoding}});
+    debug("Result of event publishing: ", response.data);
+    expect(response.data).to.have.nested.property("order");
```

Comment:
Why are we checking these details in the response here? Shouldn't we just check that we got a 204 from publisher proxy, and that would be enough?

Reply:
I check now for the 200 status